etc/kube/cron_deployment_template.yaml

```diff
@@ -0,0 +1,61 @@
+# Kubernetes
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: "${CRON_DEPLOYMENT_NAME}"
+spec:
+  replicas: 1
+  minReadySeconds: 5
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: "${CRON_DEPLOYMENT_APP}"
+    spec:
+      volumes:
+      - name: stackage-server-cron-volume
+        secret:
+          secretName: ${CRON_DEPLOYMENT_NAME}-secret
+      containers:
+      - name: stackage-server-cron
+        image: "${DEPLOYMENT_IMAGE}"
+        imagePullPolicy: Always
+        env:
+        - name: PGSTRING
+          valueFrom:
+            secretKeyRef:
+              name: "${DEPLOYMENT_NAME}"
+              key: PGCONN
+        command:
+        - bash
+        - "-c"
+        - |
+          set -eux
+          source /secret/aws.sh
+          mkdir -p /work
+          cd /work
+          while true
```

Comment:
Could this be turnedinto an actual CronJob rather than a Deployment with a bash loop?